src/utils/urlJoin.js

```diff
@@ -6,7 +6,7 @@ const path = require('path');
  * e.g. from \path\to\res.js to /path/to/res.js.
  */
 module.exports = function(publicURL, assetPath) {
-  const url = URL.parse(publicURL);
+  const url = URL.parse(publicURL, false, true);
```

Comment:
[Documentation](https://nodejs.org/api/url.html#url_url_parse_urlstring_parsequerystring_slashesdenotehost)

- 2nd param is `parseQueryString` which already defaults to false. 
- 3rd param is `slashesDenoteHost` which this changes to `true`